pipelines/docker-build-multi-platform-oci-ta/README.md

```diff
@@ -61,6 +63,8 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 |LABELS| Additional key=value labels that should be applied to the image| []| |
 |PLATFORM| The platform to build on| None| |
 |PREFETCH_INPUT| In case it is not empty, the prefetched content should be made available to the build.| | '$(params.prefetch-input)'|
+|PRIVILEGED| Whether to add --privileged to podman run which runs remotely| false| |
```

Comment:
Here's my proposal: let's consolidate all `PRIVILEGED`, `SECURITY_OPTS` and `DEVICES` to just: `PRIVILEGED_NESTED` or something.

It's totally fine if e.g. `--privileged` automatically implices `--devices /dev/fuse` - there is no loss of generality here; privileged code can use fuse anyways.

I think it's OK if `PRIVILEGED_NESTED` implies `--security-opts=label=disable` too.

Reply:
Yes, that's a good idea, but I saw in previous comments,  @brianwcook suggest to use this PR to replace https://github.com/konflux-ci/build-definitions/pull/1529/files , where they use `--security-opts` and `--devices` separately and they don't use `--privileged`.

I add `--devices` and `--security-opts` along with `--privileged` is because the outermost `podman run` could just use `--privileged` but the inner `buildah build` doesn't accept `--privileged` and I need to pass  `--devices` and `--security-opts` to it, do you have any suggests for this?

Reply:
Merged into a single `PRIVILEGED_NESTED` now.